_episodes/02-starting-with-data.md

```diff
@@ -1,73 +1,75 @@
+    - "Explicar qué es una biblioteca y para qué se usa."
+    - "Describir qué es, Pandas, la Biblioteca de Análisis de Datos de Python."
+    - "Cargar Pandas, la biblioteca de Análisis de Datos de Python."
+    - "Usar `read_csv` para leer datos tabulares en Python."
+    - "Describir qué es un **DataFrame** en Python."
+    - "Acceder y resumir datos guardados en un **DataFrame**."
+    - "Definir la indexación y su relación con las estructuras de datos."
+    - "Ejecutar operaciones matemáticas básicas y calcular estadísticas descriptivas sobre datos dentro de un Pandas **Dataframe**."
+    - "Crear gráficos simples."
 keypoints:
     - "FIXME"
 ---
 
-# Working With Pandas DataFrames in Python
+# Trabajando con Pandas **DataFrames** en Python
 
-We can automate the process of performing data manipulations in Python. It's efficient to spend time
-building the code to perform these tasks because once it's built, we can use it
-over and over on different datasets that use a similar format. This makes our
-methods easily reproducible. We can also easily share our code with colleagues
-and they can replicate the same analysis.
+Podemos automatizar el proceso de manipular datos con Python. Vale la pena pasar
+tiempo escribiendo el código que haga estas tareas ya que una vez que se escribió,
+lo podemos usar una y otra vez en distintos conjuntos de datos que usen un formato 
+similar. Esto hace nuestros métodos fácilmente reproducibles. También resulta fácil
+compartir nuestro código con nuestros colegas y ellos pueden replicar el mismo 
+análisis.
 
-### Starting in the same spot
+### Comenzando en el mismo lugar
 
-To help the lesson run smoothly, let's ensure everyone is in the same directory.
-This should help us avoid path and file name issues. At this time please
-navigate to the workshop directory. If you working in IPython Notebook be sure
-that you start your notebook in the workshop directory.
+Para que la lección salga de la mejor manera posible, vamos a asegurarnos que todos
+estemos en el mismo directorio. Esto debería ayudarnos a evitar inconvenientes con los nombres
+de los directorios y los archivos. Navega al directorio del
+taller. Si tu estás trabajando en IPython Notebook asegurate de iniciar tu notebook 
+en el directorio del taller.
 
-A quick aside that there are Python libraries like [OS
-Library](https://docs.python.org/3/library/os.html) that can work with our
-directory structure, however, that is not our focus today.
+Un comentario al margen es que hay bibliotecas de Python, como [OS
+Library](https://docs.python.org/3/library/os.html), las cuales pueden trabajar con la
+estructra de directorios, sin emebargo este no es nuestro objetivo el día de hoy. 
 
-### Our Data
+### Nuestros datos
 
-For this lesson, we will be using the Portal Teaching data, a subset of the data
-from Ernst et al
+Para esta lección, usaremos los datos del portal de enseñanza, estos datos son un
+subconjunto de Ernst et al
```

Comment:
Agrego un par de palabras y un punto a et al. de acuerdo a las [normas APA](https://blog.apastyle.org/apastyle/2011/02/et-al-when-and-how.html). 
```suggestion
subconjunto de los datos estudiados por Ernst et al.
```